VMBackup/main/freezesnapshotter.py

```diff
@@ -136,12 +140,23 @@ def update_snapshotinfoarray(self, blob_snapshot_info_array):
 
         return snapshot_info_array
 
-    def updateErrorCode(self, blob_snapshot_info_array, all_failed):
+    def updateErrorCode(self, blob_snapshot_info_array, all_failed, unable_to_sleep, is_inconsistent):
         run_result = CommonVariables.success
         any_failed = False
         run_status = 'success'
 
-        if blob_snapshot_info_array != None:
+        if unable_to_sleep:
+            run_result = CommonVariables.error
+            run_status = 'error'
+            error_msg = 'T:S Machine unable to sleep'
+            self.logger.log(error_msg, True, 'Error')
+        elif is_inconsistent == True :
+            run_result = CommonVariables.error
+            run_status = 'error'
+            error_msg = 'Snapshots are inconsistent'
+            self.logger.log(error_msg, True, 'Error')
+
```

Comment:
remove line-gap (just to avoid any unknown effect on some python version)

Reply:
will remove the line gap between elif statements